cmd/litcli/autopilot.go

```diff
@@ -51,20 +56,29 @@ var addAutopilotSessionCmd = cli.Command{
 			Name:     "feature",
 			Required: true,
 		},
-		cli.StringFlag{
-			Name: "channel-restrict-list",
+		cli.StringSliceFlag{
+			Name: "feature-channel-restrict-list",
 			Usage: "list of channel IDs that the " +
 				"Autopilot server should not " +
 				"perform actions on. In the " +
 				"form of: chanID1,chanID2,...",
 		},
-		cli.StringFlag{
-			Name: "peer-restrict-list",
+		cli.StringSliceFlag{
+			Name: "feature-peer-restrict-list",
```

Comment:
note that there's an issue with this flag and the privacy mapper, if repeated with the same pubkey, the mapping fails. will fix this in the next round

Reply:
this is going to be fixed in https://github.com/lightninglabs/lightning-terminal/pull/634